README.md

```diff
@@ -10,53 +10,57 @@
 
 # Documentation
 
-Checkout the [user documentation](https://nanopub.readthedocs.io/)
+Checkout the **[user documentation 📖 ](https://fair-workflows.github.io/nanopub)**
 
 # Setup
 Install using pip:
 ```
 pip install nanopub
 ```
 
-To publish to the nanopub server you need to setup your profile. This allows the nanopub server to identify you. Run 
-the following command in the terminal:
+To publish to the nanopub server you need to setup your profile. This allows the nanopub server to identify you. Run the following command in the terminal:
 ```
-setup_nanopub_profile
+np setup
 ```
-This will ask you a few questions, then it will use that information to add and store RSA keys to sign your nanopublications with, (optionally) publish a nanopublication with your name and ORCID iD to declare that you are using these RSA keys, and store your ORCID iD to automatically add as author to the
-provenance of any nanopublication you will publish using this library.
+This will ask you a few questions, then it will use that information to add and store RSA keys to sign your nanopublications with, (optionally) publish a nanopublication with your name and ORCID iD to declare that you are using these RSA keys, and store your ORCID iD to automatically add as author to the provenance of any nanopublication you will publish using this library.
 
 ## Quick Start
 
 
 ### Publishing nanopublications
 ```python
-
-from nanopub import Publication, NanopubClient
-from rdflib import Graph, URIRef, RDF, FOAF
-
-# Create the client, that allows searching, fetching and publishing nanopubs
-client = NanopubClient()
-
-# Either quickly publish a statement to the server
-client.claim('All cats are gray')
-
-# Or: 1. construct a desired assertion (a graph of RDF triples)
+from rdflib import Graph
+from nanopub import Nanopub, NanopubConf, load_profile
+
+# 1. Create the config
+np_conf = NanopubConf(
+    use_test_server=True,
+    profile=load_profile(),
+    add_prov_generated_time=True,
+    attribute_publication_to_profile=True,
+)
+
+# 2. Construct a desired assertion (a graph of RDF triples) using rdflib
 my_assertion = Graph()
-my_assertion.add( (URIRef('www.example.org/timbernerslee'), RDF.type, FOAF.Person) )
-
-# 2. Make a Publication object with this assertion
-publication = Publication.from_assertion(assertion_rdf=my_assertion)
-
-# 3. Publish the Publication object. The URI at which it is published is returned.
-publication_info = client.publish(publication)
-print(publication_info)
+my_assertion.add((
+    rdflib.URIRef('www.example.org/timbernerslee'),
+    rdflib.RDF.type,
+    rdflib.FOAF.Person
+))
+
+# 2. Make a Nanopub object with this assertion
+np = Nanopub(
+    conf=np_conf,
+    assertion=my_assertion
+)
+
+# 3. Publish the Nanopub object
+np.publish()
```

Comment:
Yes no need for the client when publishing! 
Publishing is now directly implemented in the `Nanopub` object, it was cleanier, and worked better when creating custom nanopub class by inheritance (so you can override it if needed), and allowed to pass the Configuration more easily accross the different object (having the conf params directly in the client would cause circular dependencies)

The client still exists just for searching nanopubs, it's basically just a client for the GRLC API (fetching has also been implemented directly in the `Nanopub` object). I did not change much there, we could maybe rename the class for more clarity
You can find the docs for using the client here: https://vemonet.github.io/nanopub/searching/searching

Reply:
OK this makes perfect sense, thank you. I agree that how you've split it makes sense. I also wonder if we can rename the client since it is just a wrapper around GRLC API calls. For now let's just leave it as it is